webpack/reducers/reducer_generator.js

```diff
@@ -0,0 +1,37 @@
+import _ from 'lodash';
+
```

Comment:
Ok, so some of the files are a general refactoring along these lines and that is why they were included in this PR, right?